### ISSUE TITLE
0.1.1 Helm chart deployment fails with default settings

### ISSUE THREAD

**Issue by apogrebnyak** [jenkinsci/kubernetes-operator#338]
In 0.1.0 helm chart was refactored to include an automatic setup of `backup` container.

When the app was upgraded to 0.4.0 the `backup` container is now failing to start.
When it fails to start it also brings down the `jenkins-master`.

You see the following repeated lines in operator log:
```
2020-04-19T12:35:35.811Z	INFO	controller-jenkins	base/reconcile.go:499	Creating a new Jenkins Master Pod central-jenkins-alextest/jenkins-jenkins	{"cr": "jenkins"}
2020-04-19T12:36:00.616Z	INFO	controller-jenkins	base/reconcile.go:926	Container 'backup' is terminated, status '{Name:backup State:{Waiting:nil Running:nil Terminated:&ContainerStateTerminated{ExitCode:1,Signal:0,Reason:Error,Message:,StartedAt:2020-04-19 12:35:49 +0000 UTC,FinishedAt:2020-04-19 12:35:59 +0000 UTC,ContainerID:docker://33e70ef965f495fcd3b5046e96af59bf60cd4e0672c9bf58b456e8a92f0786fb,}} LastTerminationState:{Waiting:nil Running:nil Terminated:nil} Ready:false RestartCount:0 Image:virtuslab/jenkins-operator-backup-pvc:v0.0.8 ImageID:docker-pullable://virtuslab/jenkins-operator-backup-pvc@sha256:3941f2934347e929577219bcc95672937696d0e1a56622b2bc92e2c12b74698f ContainerID:docker://33e70ef965f495fcd3b5046e96af59bf60cd4e0672c9bf58b456e8a92f0786fb Started:<nil>}'	{"cr": "jenkins"}
2020-04-19T12:36:44.400Z	INFO	controller-jenkins	base/reconcile.go:499	Creating a new Jenkins Master Pod central-jenkins-alextest/jenkins-jenkins	{"cr": "jenkins"}
2020-04-19T12:37:14.251Z	INFO	controller-jenkins	base/reconcile.go:926	Container 'backup' is terminated, status '{Name:backup State:{Waiting:nil Running:nil Terminated:&ContainerStateTerminated{ExitCode:1,Signal:0,Reason:Error,Message:,StartedAt:2020-04-19 12:37:03 +0000 UTC,FinishedAt:2020-04-19 12:37:13 +0000 UTC,ContainerID:docker://a0705cb117b90c9762b28b9463a15e8d16e7863deb4572927a25a4bf8799d90d,}} LastTerminationState:{Waiting:nil Running:nil Terminated:nil} Ready:false RestartCount:0 Image:virtuslab/jenkins-operator-backup-pvc:v0.0.8 ImageID:docker-pullable://virtuslab/jenkins-operator-backup-pvc@sha256:3941f2934347e929577219bcc95672937696d0e1a56622b2bc92e2c12b74698f ContainerID:docker://a0705cb117b90c9762b28b9463a15e8d16e7863deb4572927a25a4bf8799d90d Started:<nil>}'	{"cr": "jenkins"}
```

The backup container has this in its log:
```
Trimming to only 3 recent backups in preparation for new backup
find: '/backup/lost+found': Permission denied
```

Adding this to here -> https://github.com/jenkinsci/kubernetes-operator/blob/master/chart/jenkins-operator/templates/jenkins.yaml#L111
```
    securityContext:
      runAsUser: 1000
      fsGroup: 1000
```
fixes the problem with backup container.

This was first described here -> https://github.com/jenkinsci/kubernetes-operator/issues/333#issuecomment-616124896

**Comment by tomaszsek** [comment]
Fixed in `0.1.2` Helm Chart

**Comment by apogrebnyak** [comment]
For whatever reason all the comments are gone from the `values.yaml` shipped in tarball.
